estate/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+estate.access_estate_property,access_estate_property,estate.model_estate_property,base.group_user,1,0,0,0
```

Comment:
It is too early to worry about access rights, you can safely give all permissions to the user group for now and reconsider later